list.go

```diff
@@ -10,12 +10,25 @@ import (
 	"text/tabwriter"
 	"time"
 
+	"encoding/json"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/codegangsta/cli"
 	"github.com/opencontainers/runc/libcontainer"
 )
 
-const formatOptions = `table, json, yaml, xml, or "go=<go-template text>"`
+const formatOptions = `table or json`
+
+type containerList struct {
+	ContainerStates []*containerState `json:"containerlist"`
+}
+
+// containerState represents the platform agnostic pieces relating to a
+// running container's status and state
+type containerState struct {
+	Status string `json:"status"`
+	libcontainer.BaseState
```

Comment:
Don't use the internal libcontainer types here.  recreate only the information that you want consumable via the public api.  We don't want to make a change on an internal type and then have to break a consumer because it is being used here.